### ISSUE TITLE
Safe configuration of web root directory

### ISSUE THREAD

**Issue by LukeZS** [glpi-project/glpi#14488]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.7

### Bug description

The following message is displayed after upgrading from 10.0.6 to 10.0.7
Web server root directory configuration is not safe as it permits access to non-public files. See installation documentation for more details.
Web server root directory should be C:\xampp\htdocs\helpdesk\public to ensure non-public files cannot be accessed.

The slashes should be the opposites: C:/xampp/htdocs/helpdesk/public

Anyway I've edited the file:
C:\xampp\apache\conf\httpd.conf

Changing this:

DocumentRoot "C:/xampp/htdocs"
<Directory "C:/xampp/htdocs">

With that:

DocumentRoot "C:/xampp/htdocs/helpdesk/public"
<Directory "C:/xampp/htdocs/helpdesk/public">

Apache re-started correctly, but the glpi page could not be opened (not found).

### Relevant log output

```shell
No relevant log output found.
```

### Steps To reproduce

Open glpi with and administrative user.

### Your GLPI setup information

GLPI 10.0.7 (/helpdesk => C:\xampp\htdocs\helpdesk)
Installation mode: TARBALL
Current language:en_GB

Operating system: Windows NT ASSET2 6.3 build 9600 (Windows Server 2012 R2 Standard Edition) AMD64
PHP 8.1.5 apache2handler (Core, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apache2handler, bcmath, bz2, calendar,
	ctype, curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, intl, json, ldap, libxml, mbstring, mysqli,
	mysqlnd, openssl, pcre, pdo_mysql, pdo_sqlite, readline, session, sodium, standard, tokenizer, xml, xmlreader, xmlwriter, zip,
	zlib)
Setup: max_execution_time="1200" memory_limit="512M" post_max_size="512M" safe_mode="" session.save_handler="files"
	upload_max_filesize="400M" 
Software: Apache/2.4.53 (Win64) OpenSSL/1.1.1n PHP/8.1.5 (Apache/2.4.53 (Win64) OpenSSL/1.1.1n PHP/8.1.5 Server at
	asset.rgdomain.local Port 80
)
	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36
Server Software: mariadb.org binary distribution
	Server Version: 10.4.24-MariaDB
	Server SQL Mode: NO_ZERO_IN_DATE,NO_ZERO_DATE,NO_ENGINE_SUBSTITUTION
	Parameters: root@localhost/glpi
	Host info: localhost via TCP/IP
	
PHP version (8.1.5) is supported.PHP version (8.1.5) is supported.
Sessions configuration is OK.Sessions configuration is OK.
Allocated memory is sufficient.Allocated memory is sufficient.
mysqli extension is installed.mysqli extension is installed.
Following extensions are installed: dom, fileinfo, json, simplexml.Following extensions are installed: dom, fileinfo, json, simplexml.
curl extension is installed.curl extension is installed.
gd extension is installed.gd extension is installed.
intl extension is installed.intl extension is installed.
libxml extension is installed.libxml extension is installed.
zlib extension is installed.zlib extension is installed.
The constant SODIUM_CRYPTO_AEAD_XCHACHA20POLY1305_IETF_NPUBBYTES is present.The constant SODIUM_CRYPTO_AEAD_XCHACHA20POLY1305_IETF_NPUBBYTES is present.
Database engine version (10.4.24) is supported.Database engine version (10.4.24) is supported.
No files from previous GLPI version detected.No files from previous GLPI version detected.
The log file has been created successfully.The log file has been created successfully.
Write access to C:\glpi\htdocs\helpdesk\files/_cache has been validated. Write access to C:\glpi\htdocs\helpdesk\config has been validated. Write access to C:\glpi\htdocs\helpdesk\files/_cron has been validated. Write access to C:\glpi\htdocs\helpdesk\files has been validated. Write access to C:\glpi\htdocs\helpdesk\files/_dumps has been validated. Write access to C:\glpi\htdocs\helpdesk\files/_graphs has been validated. Write access to C:\glpi\htdocs\helpdesk\files/_lock has been validated. Write access to C:\glpi\htdocs\helpdesk\files/_pictures has been validated. Write access to C:\glpi\htdocs\helpdesk\files/_plugins has been validated. Write access to C:\glpi\htdocs\helpdesk\files/_rss has been validated. Write access to C:\glpi\htdocs\helpdesk\files/_sessions has been validated. Write access to C:\glpi\htdocs\helpdesk\files/_tmp has been validated. Write access to C:\glpi\htdocs\helpdesk\files/_uploads has been validated.Write access to C:\glpi\htdocs\helpdesk\files/_cache has been validated.
Write access to C:\glpi\htdocs\helpdesk\config has been validated.
Write access to C:\glpi\htdocs\helpdesk\files/_cron has been validated.
Write access to C:\glpi\htdocs\helpdesk\files has been validated.
Write access to C:\glpi\htdocs\helpdesk\files/_dumps has been validated.
Write access to C:\glpi\htdocs\helpdesk\files/_graphs has been validated.
Write access to C:\glpi\htdocs\helpdesk\files/_lock has been validated.
Write access to C:\glpi\htdocs\helpdesk\files/_pictures has been validated.
Write access to C:\glpi\htdocs\helpdesk\files/_plugins has been validated.
Write access to C:\glpi\htdocs\helpdesk\files/_rss has been validated.
Write access to C:\glpi\htdocs\helpdesk\files/_sessions has been validated.
Write access to C:\glpi\htdocs\helpdesk\files/_tmp has been validated.
Write access to C:\glpi\htdocs\helpdesk\files/_uploads has been validated.

Web server root directory configuration is not safe as it permits access to non-public files. See installation documentation for more details.Web server root directory configuration is not safe as it permits access to non-public files. See installation documentation for more details.
GLPI data directories are located in a secured path.GLPI data directories are located in a secured path.
Sessions configuration is secured.Sessions configuration is secured.
OS and PHP are relying on 64 bits integers.OS and PHP are relying on 64 bits integers.
exif extension is installed.exif extension is installed.
ldap extension is installed.ldap extension is installed.
openssl extension is installed.openssl extension is installed.
Following extensions are installed: bz2, Phar, zip.Following extensions are installed: bz2, Phar, zip.
Zend OPcache extension is installed.Zend OPcache extension is installed.
Following extensions are installed: ctype, iconv, mbstring, sodium.Following extensions are installed: ctype, iconv, mbstring, sodium.
Write access to C:\xampp\htdocs\helpdesk/marketplace has been validated.Write access to C:\xampp\htdocs\helpdesk/marketplace has been validated.
Timezones seems loaded in database.Timezones seems loaded in database.

GLPI_ROOT: "C:\\xampp\\htdocs\\helpdesk"
GLPI_CONFIG_DIR: "C:\\glpi\\htdocs\\helpdesk\\config"
GLPI_VAR_DIR: "C:\\glpi\\htdocs\\helpdesk\\files"
GLPI_MARKETPLACE_DIR: "C:\\xampp\\htdocs\\helpdesk/marketplace"
GLPI_USE_CSRF_CHECK: "1"
GLPI_CSRF_EXPIRES: "7200"
GLPI_CSRF_MAX_TOKENS: "100"
GLPI_USE_IDOR_CHECK: "1"
GLPI_IDOR_EXPIRES: "7200"
GLPI_ALLOW_IFRAME_IN_RICH_TEXT: false
GLPI_SERVERSIDE_URL_ALLOWLIST: ["/^(https?|feed):\\/\\/[^@:]+(\\/.*)?$/"]
GLPI_TELEMETRY_URI: "https://telemetry.glpi-project.org"
GLPI_INSTALL_MODE: "TARBALL"
GLPI_NETWORK_MAIL: "glpi@teclib.com"
GLPI_NETWORK_SERVICES: "https://services.glpi-network.com"
GLPI_MARKETPLACE_ALLOW_OVERRIDE: true
GLPI_MARKETPLACE_MANUAL_DOWNLOADS: true
GLPI_USER_AGENT_EXTRA_COMMENTS: ""
GLPI_DISABLE_ONLY_FULL_GROUP_BY_SQL_MODE: "1"
GLPI_AJAX_DASHBOARD: "1"
GLPI_CALDAV_IMPORT_STATE: 0
GLPI_DEMO_MODE: "0"
GLPI_CENTRAL_WARNINGS: "1"
GLPI_DOC_DIR: "C:\\glpi\\htdocs\\helpdesk\\files"
GLPI_CACHE_DIR: "C:\\glpi\\htdocs\\helpdesk\\files/_cache"
GLPI_CRON_DIR: "C:\\glpi\\htdocs\\helpdesk\\files/_cron"
GLPI_DUMP_DIR: "C:\\glpi\\htdocs\\helpdesk\\files/_dumps"
GLPI_GRAPH_DIR: "C:\\glpi\\htdocs\\helpdesk\\files/_graphs"
GLPI_LOCAL_I18N_DIR: "C:\\glpi\\htdocs\\helpdesk\\files/_locales"
GLPI_LOCK_DIR: "C:\\glpi\\htdocs\\helpdesk\\files/_lock"
GLPI_LOG_DIR: "C:\\glpi\\htdocs\\helpdesk\\files/_log"
GLPI_PICTURE_DIR: "C:\\glpi\\htdocs\\helpdesk\\files/_pictures"
GLPI_PLUGIN_DOC_DIR: "C:\\glpi\\htdocs\\helpdesk\\files/_plugins"
GLPI_RSS_DIR: "C:\\glpi\\htdocs\\helpdesk\\files/_rss"
GLPI_SESSION_DIR: "C:\\glpi\\htdocs\\helpdesk\\files/_sessions"
GLPI_TMP_DIR: "C:\\glpi\\htdocs\\helpdesk\\files/_tmp"
GLPI_UPLOAD_DIR: "C:\\glpi\\htdocs\\helpdesk\\files/_uploads"
GLPI_INVENTORY_DIR: "C:\\glpi\\htdocs\\helpdesk\\files/_inventories"
GLPI_NETWORK_REGISTRATION_API_URL: "https://services.glpi-network.com/api/registration/"
GLPI_MARKETPLACE_PLUGINS_API_URI: "https://services.glpi-network.com/api/glpi-plugins/"
GLPI_I18N_DIR: "C:\\xampp\\htdocs\\helpdesk/locales"
GLPI_VERSION: "10.0.7"
GLPI_SCHEMA_VERSION: "10.0.7@5d45269702917a32805e25b678f6779a98b145f6"
GLPI_MARKETPLACE_PRERELEASES: false
GLPI_MIN_PHP: "7.4.0"
GLPI_MAX_PHP: "8.3.0"
GLPI_YEAR: "2023"

**Comment by cedric-anne** [comment]
Hi,

Please refer to the documentation: https://glpi-install.readthedocs.io/en/develop/prerequisites.html#apache-configuration

**Comment by reinierpost** [comment]
This is a documentation bug. I just upgraded to 10.0.9 from a much older version.
I used [the upgrade instructions in the documentation](https://glpi-install.readthedocs.io/en/latest/update.html?highlight=upgrade).

GLPI works, but throws a big warning: _Web server root directory configuration is not safe as it permits access to non-public files. See installation documentation for more details._

I want to get rid of it. I go to the installation documentation. It doesn't tell me about this.
I start doing what it tells me to do. I notice the webserver configurations](https://glpi-install.readthedocs.io/en/develop/prerequisites.html#apache-configuration) tell me to use a particular site configuration for Apache. I change my site configuration to that, and as a result, GLPI no longer works.

This is broken. Either the upgrade instructions are broken, or documentation about this exists somewhere and I can't find it. Either way, the documentation should be improved.
